### PR TITLE
PLANET-6514 Restyle author archive page

### PR DESCRIPTION
**Pull request by Inwerpsel**
Ref: https://jira.greenpeace.org/browse/PLANET-6514

* Reorder mark up.
* Make it a flex layout for now.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [proteus](https://www-dev.greenpeace.org/test-proteus) | [admin](https://www-dev.greenpeace.org/test-proteus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-proteus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-proteus) | [composer-local.json](https://github.com/greenpeace/planet4-test-proteus/blob/main/composer-local.json)

:watch: 2022.02.15 15:26:48

File: assets/src/scss/pages/_author.scss [hunk 1]
```diff
@@ -1,29 +1,68 @@
...
-        overflow: hidden;
-      }
+  @include mobile-only {
+    margin: 0 auto;
+  }
 
-      img {
-        width: auto;
+  _-- {
+    max-width: 50%;
```
Comment (Inwerpsel) [inline assets/src/scss/pages/_author.scss:58]: I added these as variables just to play around with values that were not clear from the design.
Will turn this to regular CSS once the values are final.

File: templates/author.twig [hunk 2]
```diff
@@ -1,35 +1,25 @@
...
+			{% if ( author.description ) %}
+				<p class="d-md-block author-bio" aria-label="{{ __( 'Author bio', 'planet4-master-theme' ) }}">{{ fn('wpautop', fn('the_author_meta', 'description', author.id))|e('wp_kses_post')|raw }}</p>
+			{% endif %}
 		</div>
 	</header>
 
 	<div class="page-content container">
-		<h3>{{ __( 'Posts by ', 'planet4-master-theme' ) }} {{ author.name }}</h3>
+		<div class="wp-block-separator is-style-wide"></div>
+		<h2>{{ __( 'Posts by ', 'planet4-master-theme' ) }} {{ author.name }}</h2>
```
Comment (Inwerpsel) [inline templates/author.twig:22]: I made this an `h2` to increase the margin bottom as requested by Houssam. It also semantically makes more sense than `h3`.

File: templates/author.twig [hunk 3]
```diff
@@ -1,35 +1,25 @@
...
+	<header class="page-header container">
+		{% if ( author.avatar ) %}
+			<figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
+				<img itemprop="image" class="author-pic"
+					 src="{{ fn('get_avatar_url', author.id, {'size' : 300}) }}"
+					 alt="{{ author.name }}">
+			</figure>
+		{% endif %}
+		<div class="author-details">
 			<h1 class="page-header-title">{{ author.name }}</h1>
```
Comment (mleray) [inline templates/author.twig:13]: I think this `h1` tag is too big compared to the designs, at least on mobile

File: assets/src/scss/pages/_author.scss [hunk 4]
```diff
@@ -1,29 +1,68 @@
 .author {
+  .page-header {
+    display: flex;
+    justify-content: flex-start;
+    gap: 40px;
```
Comment (mleray) [inline assets/src/scss/pages/_author.scss:5]: Not a big deal, but you could use one of our spacing variables here, aka `$sp-5`

Comment (Inwerpsel) [reply assets/src/scss/pages/_author.scss:5]: I'm a bit hesitant about using those, as they make our code significantly less accessible. With using a string value of `40px`, anyone can look at a single file alone and understand what it does.

Whereas using a variable to hide these values behind a multiplication makes it impossible to understand this file without looking up the variable definition.

File: templates/author.twig (hunk 3 above)
Comment (Inwerpsel) [reply templates/author.twig:13]: I'll check this with Houssam. This `h1` is using the same style as `h1` anywhere else on the site. Not sure if it was the design's intention to use a different `h1` font size only on this page.

File: assets/src/scss/pages/_author.scss (hunk 1 above)
Comment (Inwerpsel) [reply assets/src/scss/pages/_author.scss:58]: I'll remove this once UAT is complete.

File: assets/src/scss/pages/_author.scss (hunk 4 above)
Comment (Houssam-saleh) [reply assets/src/scss/pages/_author.scss:5]: ...

File: templates/author.twig (hunk 3 above)
Comment (Inwerpsel) [reply templates/author.twig:13]: We'll keep the general `h1` size for now and maybe get back to this later.

**Pull request merged**